pkgs/development/python-modules/POT/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, numpy
+, scipy
+, cython
+, matplotlib
+, scikit-learn
+, cupy
+, pymanopt
+, autograd
+, pytestCheckHook
+, enableDimensionalityReduction ? false
+, enableGPU ? false
+}:
+
+buildPythonPackage rec {
+  pname = "POT";
```

```suggestion
  pname = "pot";
```

Please follow PEP 503 -> https://nixos.org/manual/nixpkgs/stable/#contributing

---

Thanks a lot, missed that rule for `pname`!